src/DocumentFormat.OpenXml/Packaging/OpenXmlPart.cs

```diff
@@ -141,11 +142,13 @@ internal sealed override OpenXmlPart NewPart(string relationshipType, string con
             {
                 return TargetPathOfWord;
             }
-            else if (TargetPathOfExcel is not null && package.ApplicationType == ApplicationType.Excel)
+
+            if (TargetPathOfExcel is not null && package.ApplicationType == ApplicationType.Excel)
```

Comment:
I'd like to keep the else if to group things logically that we expect it to be one or the other here

Reply:
Reinstated.